src/annotator/util/test/shadow-root-test.js

```diff
@@ -74,7 +74,23 @@ describe('annotator/util/shadow-root', () => {
       assert.notCalled(onClick);
     });
 
-    it('stops propagation of touchstart events', () => {
+    it('stops propagation of "mousedown" events', () => {
```

Comment:
Probably a good test to add, looks like no code changes needed to shadow-root as it already does this.

Reply:
I added the code in the shadow-root to stop the bubbling up of the `mousedown`, otherwise all clicks in `<hypothesis-....>` will cause the sidebar to close.